packages/terra-hyperlink/src/Hyperlink.jsx

```diff
@@ -139,15 +140,18 @@ class Hyperlink extends React.Component {
       ...customProps
     } = this.props;
 
-    const hyperlinkClasses = cx([
+    const theme = this.context;
+
+    const hyperlinkClasses = cx(
       'hyperlink',
       variant,
       { 'is-disabled': isDisabled },
       { 'is-underline-hidden': isUnderlineHidden },
       { 'is-active': this.state.active },
       { 'is-focused': this.state.focused },
+      theme.className,
       customProps.className,
```

Comment:
```suggestion
    const hyperlinkClasses = classNames(
      cx(
        'hyperlink',
        variant,
        { 'is-disabled': isDisabled },
        { 'is-underline-hidden': isUnderlineHidden },
        { 'is-active': this.state.active },
        { 'is-focused': this.state.focused },
        theme.className,
      ),
      customProps.className,
```

Reply:
I had to put`customProps.className` into bind function due to [this test](https://github.com/cerner/terra-core/blob/master/packages/terra-hyperlink/src/terra-dev-site/test/hyperlink/StatesHyperlink.test.jsx).
As it requires default state styles from hyperlink CSS. 
Should I pull in the styles needed into common CSS for the tests?

Reply:
Updated here https://github.com/cerner/terra-core/pull/2989/commits/9317344b9755a52e5484ea87936bebbaccefb4e6.